### ISSUE TITLE
[Programmatic Access – Make Code - Home page] Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content (html)

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3523]
**User Experience**:
On any page, if the landmarks are not properly defined then the users who depends upon Screen Reader for navigation might get affected and failed to navigate properly on the page.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Run accessibility insights for web and select fast pass.
5. Verify the issue.

**Actual Result:**
Each page does not contains one mechanism for a user to bypass navigation and jump straight to the content.

**Expected Result:**
Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content.

**Issue**: Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content (bypass - https://dequeuniversity.com/rules/axe/4.0/bypass?application=msftAI)

**Target application**: Microsoft MakeCode for micro:bit - https://makecode.microbit.org/#

**Element path**: html

**Snippet:** <html lang="en" manifest="/---manifest" data-framework="typescript">

**How to fix:**
Fix any of the following:
  No valid skip link found
  Page does not have a heading
  Page does not have a landmark region

This accessibility issue was found using Accessibility Insights for Web 2.22.0 (axe-core 4.0.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**WCAG Reference:**
[WCAG 2.4.1 - Bypass Blocks](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA
Tool: Accessibility insights for web

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.1, #Win10-Edge(Chromium)

![WCAG2 4 1_Ensure each page has at least one mechanism](https://user-images.githubusercontent.com/72871089/97665833-67987180-1aa2-11eb-9ecb-29c67c3ca5a2.jpg)

**Comment by jwunderl** [comment]
No repro in accessibility insights / we have a tab menu with options for different jumps: 
![image](https://user-images.githubusercontent.com/5615930/113215224-645f0880-922f-11eb-9612-b037dfdb97b3.png)
